docu/files/ExtModules.html.md

```diff
@@ -837,16 +836,66 @@ Use the context variable `currentLanguage` to determine the language to which th
     </tr>
 </table>
 
-Based on the information that this variable provides, developers select localized content that they have previously configured using the technology of their choice.
+Use the context variable `currentLanguage` to determine the current language set for the Builder module. Based on the information that this variable provides, developers select localized content that they have previously configured using the technology of their choice.<br><br>
```

Comment:
Format variable and parameter names (as opposed to their values) in bold, not code format: **currentLanguage**. The value of **currentLanguage** would be in code format: `EN` or `DE`.